README.md

```diff
@@ -23,121 +23,142 @@ Sourcebot is a fast code indexing and search tool for your codebases. It is buil
+### (Optional) Provide an access token to index private repositories
+In order to allow Sourcebot to index your private repositories, you must provide it with an access token.
 
-    <div>
-    <details open>
-    <summary>
-    <picture>
-        <source media="(prefers-color-scheme: dark)" srcset=".github/images/github-favicon-inverted.png">
-        <img src="https://github.com/favicon.ico" width="16" height="16" alt="GitHub icon">
-    </picture>
-    GitHub
-    </summary>
+<div>
+<details>
+<summary><img src="https://github.com/favicon.ico" width="16" height="16" /> GitHub</summary>
 
-    Run the `sourcebot` docker image, passing in the Github PAT you generated in the previous step as an environment variable called `GITHUB_TOKEN`:
-    ```sh
-    docker run -p 3000:3000 --rm --name sourcebot -v $(pwd):/data -e GITHUB_TOKEN=$GITHUB_TOKEN ghcr.io/taqlaai/sourcebot:main
-    ```
-    </details>
+Generate a GitHub Personal Access Token (PAT) [here](https://github.com/settings/tokens/new) and make sure you select the `repo` scope.
 
-    <details>
-    <summary><img src="https://gitlab.com/favicon.ico" width="16" height="16" /> GitLab</summary>
+You'll need to pass this PAT each time you run Sourcebot by setting the GITHUB_TOKEN environment variable:
 
-    ```sh
-    docker run -p 3000:3000 --rm --name sourcebot -v $(pwd):/data -e GITLAB_TOKEN=<token> ghcr.io/taqlaai/sourcebot:main
-    ```
+<pre>
+docker run -p 3000:3000 --rm --name sourcebot -e <b>GITHUB_TOKEN=[your-github-token]</b> -v $(pwd):/data ghcr.io/taqlaai/sourcebot:main
+</pre>
 
-    </details>
+</details>
 
-    <details>
-    <summary><img src="https://bitbucket.org/favicon.ico" width="16" height="16" /> BitBucket</summary>
+<details>
+<summary><img src="https://gitlab.com/favicon.ico" width="16" height="16" /> GitLab</summary>
 
-    TODO
+TODO
 
-    </details>
-    </div>
+</details>
 
-    Two things should happen: (1) a `.sourcebot` directory will be created containing the mirror repositories and indexes, and (2) you will see output similar to:
+<details>
+<summary><img src="https://bitbucket.org/favicon.ico" width="16" height="16" /> BitBucket</summary>
 
-    ```sh
-    INFO spawned: 'node-server' with pid 10
-    INFO spawned: 'zoekt-indexserver' with pid 11
-    INFO spawned: 'zoekt-webserver' with pid 12
-    run [zoekt-mirror-github -dest /data/.sourcebot/repos -delete -org <org>]
-    ...
-    INFO success: node-server entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
-    INFO success: zoekt-indexserver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
-    INFO success: zoekt-webserver entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
-    ```
+TODO
 
-    zoekt will now index your repositories (at `HEAD`). By default, it will re-index existing repositories every hour, and discover new repositories every 24 hours.
+</details>
+</div>
+
+### (Optional) Specifying a config file
```

Comment:
Thoughts on having  `-e CONFIG_PATH=...` explicitly defined in the previous commands and not needing this section?